### ISSUE TITLE
FR: Control output naming

### ISSUE THREAD

**Issue by gerroon** [seblucas/cops#355]
Hi

It would be very nice if one could control the file naming of the downloads. I would like to keep the naming same across my different setups. Basically I want to be able to control how the saved/downloaded files are named.

**Comment by ubupl** [comment]
+1

**Comment by seblucas** [comment]
@gerroon 
IIRC Calibre has a setting for that, do you use it ?

**Comment by gerroon** [comment]
Well I use the device settings to control how it is saved. Does Cops respect any of the Calibre settings?

**Comment by seblucas** [comment]
I should have been clearer, no COPS does not handle it for now.

I just wanted to check that there is a setting in Calibre to do it, that should be the way to do it in COPS too.

**Comment by gerroon** [comment]
Yeah you can set how the names are saved per device and I think there isa  globa; way to do it too.


Thnx
